app/src/main/res/values/dimens.xml

```diff
@@ -1,770 +1,668 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-  <!-- Default screen margins, per the Android Design guidelines. -->
-  <dimen name="activity_horizontal_margin">16dp</dimen>
-  <dimen name="activity_vertical_margin">16dp</dimen>
-  <dimen name="match_parent">-2px</dimen>
-  <dimen name="answer_tick_size">24dp</dimen>
-  <dimen name="answer_tick_margin">12dp</dimen>
-  <integer name="fade_in_duration_ms">1000</integer>
-  <integer name="fade_out_duration_ms">1000</integer>
-  <dimen name="audio_fragment_corner_radius">8dp</dimen>
-  <dimen name="audio_fragment_margin">28dp</dimen>
-  <dimen name="cellular_data_dialog_padding">24dp</dimen>
-  <dimen name="state_previous_next_button_radius">4dp</dimen>
-  <dimen name="train_checkbox_padding">18dp</dimen>
-  <dimen name="divider_margin_top">8dp</dimen>
-  <dimen name="divider_margin_bottom">8dp</dimen>
-  <dimen name="bottom_white_space">160dp</dimen>
-  <dimen name="text_input_layout_corner_radius">4dp</dimen>
-  <dimen name="text_input_layout_stroke_width">1dp</dimen>
-
-  <dimen name="clickable_item_min_height">48dp</dimen>
-
-  <dimen name="ongoing_story_card_view_is_tablet_margin_top">28dp</dimen>
-  <dimen name="ongoing_story_card_view_margin_top_portrait_story_grid_greater_then_2">28dp</dimen>
-  <dimen name="ongoing_story_card_view_margin_top_landscape">28dp</dimen>
-  <dimen name="font_scale_content_extra_large_text_view_size">28dp</dimen>
-  <dimen name="state_fragment_non_split_view_margin_end">28dp</dimen>
-  <dimen name="state_fragment_non_split_view_margin_start">28dp</dimen>
+  <!--Questions-->
```

Comment:
Please arrange the comments and their respective Dimens in AtoZ order. So that it's more clear.

Reply:
Ok, But I was following this  [](https://github.com/oppia/oppia-android/pull/2111#discussion_r523663618)

Reply:
![image](https://user-images.githubusercontent.com/95977604/212160506-24c523e9-231c-4c46-9f2d-6f0b8f7a5ac7.png)

Reply:
should I still do it Alphabetically ?

Reply:
@BenHenning what you suggest we need to arrange Dimes https://github.com/oppia/oppia-android/pull/4818#discussion_r1068508897 as per this?

Reply:
Thanks @BenHenning for the clarification.
@Yashsomalkar PTAL on Ben's comment https://github.com/oppia/oppia-android/pull/4818#discussion_r1069948304.